cpp/CMakeLists.txt

```diff
@@ -534,19 +536,73 @@ if(BUILD_CUML_CPP_LIBRARY)
       $<INSTALL_INTERFACE:include>
   )
 
+  set(_cuml_cpp_public_libs)
+  set(_cuml_cpp_private_libs)
```

Comment:
why this format for naming these variables? I would expect something more like `CUML_CPP_PUBLIC_LIBS`

Reply:
Just a convention for private variable naming. Uppercase can be confused for variables/options that are allowed to be set externally.